hole/__init__.py

```diff
@@ -49,6 +50,48 @@ def __init__(self, host, loop, session, location='admin', tls=False,
             _LOGGER.error(msg)
             raise exceptions.HoleConnectionError(msg)
 
+    async def enable(self):
+        """Enable DNS blocking on a *hole instance."""
+        if self.api_token is None:
+            _LOGGER.error("You need to supply an api_token to use this")
+            return
+        api = "{}?enable=True&auth={}".format(self.base_url, self.api_token)
+        try:
+            async with async_timeout.timeout(5, loop=self._loop):
+                response = await self._session.get(api)
+
+            _LOGGER.info(
+                "Response from *hole: %s", response.status)
+            data = await response.json()
+            _LOGGER.debug(data)
+
+        except (asyncio.TimeoutError, aiohttp.ClientError, socket.gaierror):
+            msg = "Can not load data from *hole: {}".format(self.host)
+            _LOGGER.error(msg)
+            raise exceptions.HoleConnectionError(msg)
+
+
+    async def disable(self, duration=True):
+        """Disable DNS blocking on a *hole instance."""
+        if self.api_token is None:
+            _LOGGER.error("You need to supply an api_token to use this")
+            return
+        api = "{}?disable={}&auth={}".format(self.base_url, duration,
```

Comment:
Can we use `params` here? Would be simpler to read.

Reply:
Moved params to params now :)